### DISCUSSION TITLE
Any ambitions to implement static comma vectors?

### DISCUSSION THREAD

**Discussion by D3lta-2-1** [bitshifter/glam-rs#525] [Ideas]
Static point vectors would be really handy (thanks to the “Fixed” crate). They're not native types, but they implement all the features needed for arithmetic. Can you consider this as an optional feature for Glam ?

__Fixed__: https://crates.io/crates/fixed

**Comment by bitshifter** [comment]
Interesting crate. Realistically I don't have the time to commit to doing that. Even accepting it as a patch might be a bit too much of a commitment given fixed point is not something I have any experience with so addressing any issues would be challenging.

If someone wanted to give it a go I can help support the effort. It would probably make more sense to be a separate crate anyway but perhaps the glam code generator could be made more shareable for such a task.
